### PR TITLE
Feature/76 select from values as

### LINKED ISSUE #77: Add support for SELECT FROM (SELECT ...) queries
Add support for selecting from sub selects.

For example:
```SELECT * FROM (SELECT * FROM ...)```

Until now the sql-statement-builder has no support for adding subselcets to the from clause.

### PR DESCRIPTION
**Pull request by AnastasiiaSergienko**
Closes #77

### CODE REVIEW

File: src/main/java/com/exasol/sql/ValueTable.java [hunk 1]
```diff
@@ -173,4 +175,45 @@ public void accept(final ValueTableVisitor visitor) {
         }
         visitor.leave(this);
     }
-}
+
+    /**
+     * Set alias for the value table.
+     * 
+     * @param tableNameAlias table name alias
+     * @param columnNamesAliases zero or more column names aliases
```
Comment (redcatbear) [inline src/main/java/com/exasol/sql/ValueTable.java:183]:
```suggestion
     * @param columnNameAliases zero or more column names aliases
```

File: src/main/java/com/exasol/sql/ValueTable.java [hunk 2]
```diff
@@ -173,4 +175,45 @@ public void accept(final ValueTableVisitor visitor) {
...
+     */
+    public void alias(final String tableNameAlias, final String... columnNamesAliases) {
```
Comment (redcatbear) [inline src/main/java/com/exasol/sql/ValueTable.java:185]:
```suggestion
    public void alias(final String tableNameAlias, final String... columnNameAliases) {
```

File: src/main/java/com/exasol/sql/ValueTable.java (hunk 1 above)
Comment (AnastasiiaSergienko) [reply src/main/java/com/exasol/sql/ValueTable.java:183]: fixed

File: src/main/java/com/exasol/sql/ValueTable.java (hunk 2 above)
Comment (AnastasiiaSergienko) [reply src/main/java/com/exasol/sql/ValueTable.java:185]: fixed

**Pull request merged**